apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -133,9 +134,9 @@ def get_objects(element, data):
             if position.shape == (3,):
                 position = position.tolist()
             else:
-                raise ValueError()
+                raise ValueError
```

Comment:
👀 this is legal? Can we provide a specific error message? Ditto elsewhere in this file.

Reply:
Resolved in [b62be5d](https://github.com/PrairieLearn/PrairieLearn/pull/11104/commits/b62be5d6c6eb998f5a4a887abe0fddc2728f445e).